__tests__/app.test.js

```diff
@@ -424,6 +412,44 @@ describe ( "app" , () => {
 
   }) ;
 
+  describe( "DELETE /api/comments/:comment_id" , () => {
+    test( "status 204: deletes the comment by comment_id" , () => {
+      
+      return request( app )
+      .delete( "/api/comments/1" )
+      .expect( 204 )
+      .then( () => {
+        return request( app )
+        .get( "/api/articles/1/comments" )
+        .expect( 200 )
```

Comment:
You don't really need to test this again as you have other tests for this endpoint, you are only really interested in the length